### PR TITLE
New version: OliverBetz.ExifTool version 13.10

### LINKED ISSUE #200346: [Update Request]: OliverBetz.ExifTool 12.97
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

OliverBetz.ExifTool

### Package Version

13.10

### Please describe the changes you would like to see

Download from https://oliverbetz.de/pages/Artikel/ExifTool-for-Windows
Several new releases have been made since the version that's still on Winget. Is there some way to automate these updates?

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #200346

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

## Manual validation

Image
Image

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/200906)

### CODE REVIEW

File: manifests/o/OliverBetz/ExifTool/13.10/OliverBetz.ExifTool.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    Custom: /CURRENTUSER
+  AppsAndFeaturesEntries:
+  - DisplayVersion: 13.10_64
+    ProductCode: ExifTool_is1
+  InstallationMetadata:
+    DefaultInstallLocation: '%LocalAppData%\Programs\ExifTool'
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://oliverbetz.de/cms/files/Artikel/ExifTool-for-Windows/ExifTool_install_13.10_32.exe
+  InstallerSha256: EC1DE44FD23992F497626DE71FD4D41E122114450D52A58C7901C3E8E7EDC4D6
```
Comment (Dragon1573) [inline manifests/o/OliverBetz/ExifTool/13.10/OliverBetz.ExifTool.installer.yaml:27]:
```suggestion
  InstallerSha256: EF2680E717A1B4C3729E931BE5C526FA1EECBA2068E77E960E74B9FD8C946036
```

File: manifests/o/OliverBetz/ExifTool/13.10/OliverBetz.ExifTool.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    Custom: /ALLUSERS
+  AppsAndFeaturesEntries:
+  - DisplayVersion: 13.10_64
+    ProductCode: ExifTool_is1
+  InstallationMetadata:
+    DefaultInstallLocation: '%ProgramFiles%\ExifTool'
+- Architecture: x86
+  Scope: machine
+  InstallerUrl: https://oliverbetz.de/cms/files/Artikel/ExifTool-for-Windows/ExifTool_install_13.10_32.exe
+  InstallerSha256: EC1DE44FD23992F497626DE71FD4D41E122114450D52A58C7901C3E8E7EDC4D6
```
Comment (Dragon1573) [inline manifests/o/OliverBetz/ExifTool/13.10/OliverBetz.ExifTool.installer.yaml:49]:
```suggestion
  InstallerSha256: EF2680E717A1B4C3729E931BE5C526FA1EECBA2068E77E960E74B9FD8C946036
```

**Review by Dragon1573 — COMMENTED** [review]
_Komac_ generated an incorrect manifest, installer file should be "re-hash".

File: manifests/o/OliverBetz/ExifTool/13.10/OliverBetz.ExifTool.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: OliverBetz.ExifTool
+PackageVersion: '13.10'
+InstallerLocale: en-US
+InstallerType: inno
+UpgradeBehavior: uninstallPrevious
+ProductCode: ExifTool_is1
+ReleaseDate: 2024-12-20
+ElevationRequirement: elevatesSelf
```
Comment (mdanish-kh) [inline manifests/o/OliverBetz/ExifTool/13.10/OliverBetz.ExifTool.installer.yaml:11]:
```suggestion
```

`Scope: user` doesn't ask for elevation.

File: manifests/o/OliverBetz/ExifTool/13.10/OliverBetz.ExifTool.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    Custom: /CURRENTUSER
+  AppsAndFeaturesEntries:
+  - DisplayVersion: 13.10_32
+    ProductCode: ExifTool_is1
+  InstallationMetadata:
+    DefaultInstallLocation: '%LocalAppData%\Programs\ExifTool'
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://oliverbetz.de/cms/files/Artikel/ExifTool-for-Windows/ExifTool_install_13.10_64.exe
+  InstallerSha256: EC1DE44FD23992F497626DE71FD4D41E122114450D52A58C7901C3E8E7EDC4D6
```
Comment (mdanish-kh) [inline manifests/o/OliverBetz/ExifTool/13.10/OliverBetz.ExifTool.installer.yaml:38]:
```suggestion
  InstallerSha256: EC1DE44FD23992F497626DE71FD4D41E122114450D52A58C7901C3E8E7EDC4D6
  ElevationRequirement: elevatesSelf
```

File: manifests/o/OliverBetz/ExifTool/13.10/OliverBetz.ExifTool.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    Custom: /ALLUSERS
+  AppsAndFeaturesEntries:
+  - DisplayVersion: 13.10_64
+    ProductCode: ExifTool_is1
+  InstallationMetadata:
+    DefaultInstallLocation: '%ProgramFiles%\ExifTool'
+- Architecture: x86
+  Scope: machine
+  InstallerUrl: https://oliverbetz.de/cms/files/Artikel/ExifTool-for-Windows/ExifTool_install_13.10_32.exe
+  InstallerSha256: EF2680E717A1B4C3729E931BE5C526FA1EECBA2068E77E960E74B9FD8C946036
```
Comment (mdanish-kh) [inline manifests/o/OliverBetz/ExifTool/13.10/OliverBetz.ExifTool.installer.yaml:49]:
```suggestion
  InstallerSha256: EF2680E717A1B4C3729E931BE5C526FA1EECBA2068E77E960E74B9FD8C946036
  ElevationRequirement: elevatesSelf
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**